planner/core/logical_plan_builder.go

```diff
@@ -3259,7 +3259,7 @@ func (b *PlanBuilder) buildUpdate(ctx context.Context, update *ast.UpdateStmt) (
 		}
 	}
 	if b.ctx.GetSessionVars().TxnCtx.IsPessimistic {
-		if !update.MultipleTable {
```

Comment:
Maybe remove these unused fields to avoid using mistakenly in the future, so dangerous

Reply:
It's in another repo, we can do it later.